From 7d92026920bd3b98a8f9e50f85647de109c36521 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anujx.mittal@intel.com>
Date: Wed, 22 Feb 2017 16:07:02 +0800
Subject: [PATCH 1/3] firefox: add .desktop to xdg autostart directory

Make sure that firefox is launched at the startup
as part of one of the requirements for the project.

Signed-off-by: Anuj Mittal <anujx.mittal@intel.com>
Signed-off-by: Chew, Chiau Ee <chiau.ee.chew@intel.com>
---
 recipes-mozilla/firefox/firefox_38.2.1esr.bbappend | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 recipes-mozilla/firefox/firefox_38.2.1esr.bbappend

diff --git a/recipes-mozilla/firefox/firefox_38.2.1esr.bbappend b/recipes-mozilla/firefox/firefox_38.2.1esr.bbappend
new file mode 100644
index 0000000..70670de
--- /dev/null
+++ b/recipes-mozilla/firefox/firefox_38.2.1esr.bbappend
@@ -0,0 +1,6 @@
+do_install_append() {
+        install -d ${D}${sysconfdir}/xdg/autostart
+        install -m 0644 ${WORKDIR}/mozilla-firefox.desktop ${D}${sysconfdir}/xdg/autostart/
+}
+
+FILES_${PN} += "${sysconfdir}/xdg/"
-- 
1.9.1

